services/ui-src/src/components/report/ReportPageWrapper.tsx

```diff
@@ -110,64 +111,66 @@ export const ReportPageWrapper = () => {
     : currentPage.elements;
   return (
     <FormProvider {...methods}>
-      <HStack width="100%" height="100%" position="relative" spacing="0">
-        {currentPage.sidebar && <Sidebar />}
-        <VStack
-          height="100%"
-          padding={
-            currentPage.sidebar
-              ? { base: "3.5rem 1rem", md: "3.5rem 0rem" }
-              : { base: "2rem 1rem", md: "2rem 0rem" }
-          }
-          margin={currentPage.sidebar ? { md: "0 4rem" } : { md: "0 6rem" }}
-          width="100%"
-          maxWidth={currentPage.sidebar ? "reportPageWidth" : "fullPageWidth"}
-          gap="0rem"
-        >
-          <Box flex="auto" alignItems="flex-start" width="100%">
-            <form
-              id="aFormId"
-              autoComplete="off"
-              onChange={handleSubmit(handleChange, handleError)}
-            >
-              {currentPage.elements && (
-                <Page elements={renderedElements ?? []}></Page>
-              )}
-            </form>
-          </Box>
-          {!currentPage.hideNavButtons && parentPage && (
-            <>
-              {parentPage.index == 0 && <Divider></Divider>}
-              <Flex width="100%" margin="1.5rem 0 0 0">
-                {parentPage.index > 0 && (
-                  <Button
-                    onClick={() => SetPageIndex(parentPage.index - 1)}
-                    variant="outline"
-                    leftIcon={<Image src={prevArrowIcon} />}
-                  >
-                    Previous
-                  </Button>
+      <ReportFormContext.Provider value={methods}>
```

Comment:
The only change in this file is that everything got wrapped inside the `<ReportFormContext.Provider value={methods}>`, everything else remained the same.